en/micro-integrator/docs/setup/observability/setting-up-minimum-basic-observability-deployment.md

```diff
@@ -30,7 +30,7 @@ Once you set up the basic deployment, you can install either one or both of the
     This provides you with Tracing capabilities. Tho use this you need to install Jaeger.
 
 
-## Setting up minimum basic observability deployment
+## Setting up minimum basic observability deployment in a virtual machine
```

Comment:
Setting up the Minimum Deployment?
Do we need 'in a virtual machine' here since we will have it in the title.

Reply:
Removed "Virtual Machine" from the title".